lib/rtcpeerconnection/edge.js

```diff
@@ -66,7 +106,12 @@ EdgeRTCPeerConnection.prototype.removeTrack = function removeTrack(sender) {
 // We need this until below PR is released
```

Comment:
@syerrapragada Can you explain a bit more here? Also add `NOTE(syerrapragada):`.